core/services/keystore/keys/ocr2key/offchain_keyring.go

```diff
@@ -47,6 +50,25 @@ func newOffchainKeyring(encryptionMaterial, signingMaterial io.Reader) (*Offchai
 	return ok, nil
 }
 
+// Decrypt decrypts a message that was encrypted using the public key
+func (ok *OffchainKeyring) Decrypt(ciphertext []byte) (plaintext []byte, err error) {
+	if len(ciphertext) < box.Overhead {
+		return nil, errors.New("ciphertext too short")
+	}
+
+	var nonce [24]byte
+	copy(nonce[:], ciphertext[:box.Overhead])
+
+	publicKey := [curve25519.PointSize]byte(ok.ConfigEncryptionPublicKey())
+
+	decrypted, success := box.OpenAnonymous(nil, ciphertext, &publicKey, &ok.encryptionKey)
```

Comment:
I though we want `Open()` and not `OpenAnonymous()`, isn't that the case? (Then `nonce` would be used.)

Reply:
I tried both, but decided to go with OpenAnonymous as I couldn't see how the verification would be necessary.  (`nonce` was an artifact I left by mistake).